app/viewmodels/regulatorsAndDocuments/GoverningDocumentStatusHelper.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.regulatorsAndDocuments
+
+import models.UserAnswers
+import models.regulators.SelectGoverningDocument
+import models.regulators.SelectGoverningDocument.Other
+import pages.QuestionPage
+import pages.regulatorsAndDocuments._
+import viewmodels.StatusHelper
+
+class GoverningDocumentStatusHelper extends StatusHelper {
```

Comment:
could be an object instead of a class? might help us keep it slightly more functional if we make sure it's all nice and stateless (keep vals within the function and all)